src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -683,4 +687,21 @@ protected function getAclPermissions($object)
 
         return $roles;
     }
+
+    /**
+     * @param $publicNodeVersion
+     *
+     * @return mixed
+     */
+    private function getNodeRefPage(NodeVersion $publicNodeVersion)
+    {
+        $refEntityName = $publicNodeVersion->getRefEntityName();
+
+        if (!isset($this->nodeRefs[$refEntityName])) {
+            $refEntityClass = new \ReflectionClass($refEntityName);
```

Comment:
Reflection class seems to be obsolete here. Just `new $refEntityName()` should do the trick in this case.